client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -39,13 +39,22 @@ const DesignSelector: React.FunctionComponent = () => {
 	};
 
 	const getDesignUrl = ( design: Design ) => {
+		// We temporarily show pre-generated screenshots until we can generate tall versions dynamically using mshots.
+		// See `bin/generate-gutenboarding-design-thumbnails.js` for generating screenshots.
+		// https://github.com/Automattic/mShots/issues/16
+		// https://github.com/Automattic/wp-calypso/issues/40564
+		if ( ! isEnabled( 'gutenboarding/mshot-preview' ) ) {
+			return `/calypso/page-templates/design-screenshots/${ design.slug }_${ design.template }_${ design.theme }.jpg`;
```

Comment:
I don't like that we're referencing static images directly rather than using webpack file-loader to import the images and get a generated path. That help with things like cache busting when image contents change.

Are these intended for use outside of Calypso?

This is fine to handle as a follow-up.

Reply:
Created #40708 to track.